src/styles/webheader.css

```diff
@@ -0,0 +1,17 @@
+.web-header {
+  background-color: #24243e;
+}
+
+nav {
```

Comment:
what if i'm using `<nav>` somewhere else and this style overrides the default style i want. This is why you shouldn't write class rules for elements like this unless you're sure, they class rule will be consistent everywhere.

Reply:
ok